docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
       - MAIL_DOMAIN=1024.lu
     depends_on:
       - database
-    links: 
+    links:
```

Comment:
Huh… I thought links had been deprecated.
I'd just move it under the depends_on section?

Reply:
Doesn't seem to be deprecated, https://docs.docker.com/compose/networking/#links
Though it might not be necessary